src/utils/message.js

```diff
@@ -51,3 +51,6 @@ export const shouldBeMuted = (msg, narrow, subscriptions, mutes = []): boolean =
 
   return mutes.some(x => x[0] === msg.display_recipient && x[1] === msg.subject);
 };
+
+export const isSentByYou = (message, auth): boolean =>
+  auth.email === message.sender_email;
```

Comment:
Would have suggested `isSentBySelf` as a name, but am suggesting to inline the logic.